Project.xml

```diff
@@ -3,7 +3,7 @@
 
 	<!-- _________________________ Application Settings _________________________ -->
 
-	<app title="FLTrial" file="FLTrial" main="Main" version="0.0.1" company="RafPlayz69YT" />
+	<app title="FLTSM" file="FLTSM" main="Main" version="0.0.1" company="RafPlayz69YT" />
```

Comment:
maybe new acronymn should be flftsm (fruity loops file trial state modder)

Reply:
you dont need to tho lol

Reply:
ok i guess

Reply:
domn't wanna remake the logo tho :((

Reply:
i mean you dont have to